### PR TITLE
[PW-7701] Update the Readme.md

### PR DESCRIPTION
**Pull request by AlexandrosMor**
**Description**

Update the Readme
- Add example integration for cloud terminal api 
- Add banner
- The supported endpoints are in array

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,48 +1,44 @@
-# Adyen dotnet API Library
+![NET api](https://user-images.githubusercontent.com/16090523/211498272-75a50b61-b1cc-482e-9278-c10a267efe27.png)
+# Adyen .net API Library
 [![nuget](https://img.shields.io/nuget/v/adyen.svg)](https://www.nuget.org/packages/adyen/) [![nuget](https://img.shields.io/nuget/dt/adyen.svg)](https://www.nuget.org/packages/adyen/) ![.NET Core](https://github.com/Adyen/adyen-dotnet-api-library/workflows/.NET%20Core/badge.svg)
 
-This is the officially supported dotnet library for using Adyen's APIs.
-
-## Integration
+This is the officially supported .net library for using Adyen's APIs.
```
Comment (wboereboom) [inline README.md:5]:
```suggestion
This is the officially supported .NET library for using Adyen's APIs.
```

Should we capitalise it to keep consistent?

File: README.md [hunk 2]
```diff
@@ -1,48 +1,44 @@
...
-
+```
 ## Using the library
 
-### General use with API key
-
 In order to submit http request to Adyen API you need to initialize the client. The following example makes a checkout payment request:
-~~~~ csharp
+```c#
 // Create a paymentsRequest
```
Comment (wboereboom) [inline README.md:42]: can we also add some imports?

**Review by wboereboom — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (AlexandrosMor) [reply README.md:49]: yes done

File: README.md [hunk 3]
```diff
@@ -1,48 +1,44 @@
...
-* [Platforms APIs](https://docs.adyen.com/platforms/api): Set of APIs when using Adyen for Platforms.
-  * [Hosted Onboarding API](https://docs.adyen.com/api-explorer/#/Hop/v1/overview) Current supported version: **v1**
-  * [Account API](https://docs.adyen.com/api-explorer/#/Account/v5/overview) Current supported version: **v5**
-  * [Fund API](https://docs.adyen.com/api-explorer/#/Fund/v5/overview) Current supported version: **v5**
-* [Cloud-based Terminal API](https://docs.adyen.com/point-of-sale/choose-your-architecture/cloud): Our point-of-sale integration.
-* [Local-based Terminal API](https://docs.adyen.com/point-of-sale/choose-your-architecture/local): Our point-of-sale integration.
-* [POS Terminal Management API](https://docs.adyen.com/api-explorer/#/postfmapi/v1/overview): Endpoints for managing your point-of-sale payment terminals **v1**
+|API | Description | Service Name | Supported version |
+|----| ----------- |-------|--------|
+| [Checkout API](https://docs.adyen.com/api-explorer/#/CheckoutService/v68/overview) | Our latest integration for accepting online payments. Current supported version | Checkout API | **v69** |
```
[the remark this answers is not in the document]
Comment (AlexandrosMor) [reply README.md]: I used the terminology from the docs

**Review by jillingk — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1,92 +1,163 @@
-# Adyen dotnet API Library
+![NET api](https://user-images.githubusercontent.com/16090523/211498272-75a50b61-b1cc-482e-9278-c10a267efe27.png)
+# Adyen .net API Library
```
Comment (AlexandrosMor) [inline README.md:2]:
```suggestion
# Adyen .NET API Library
```

**Review by jillingk — APPROVED** [review]

**Comment by wboereboom** [comment]
I broke the README by fixing the merge conflict. Mea Culpa.

**Review by wboereboom — APPROVED** [review]

**Review by jillingk — APPROVED** [review]

**Pull request merged**